examples/turn-server/simple/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM golang:1.10
```

Comment:
How about we use a more recent Go version?
As far as I am aware those old versions are EOL.
Do they are even still receive security updates?

Reply:
used latest instead